CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[1.0.1] - 2016-10-14
+~~~~~~~~~~~~~~~~~~~~
+
+* Fixed two instances of line-height being slightly increased in an ugly way.
```

Comment:
Technically, according to http://keepachangelog.com/, this bullet item should be nested under a "Changed" sub-header.  Not sure how much we really care about that, but it's the closest thing I could find to a reasonable standard for writing changelogs.